homeassistant/components/qnap/coordinator.py

```diff
@@ -36,9 +36,11 @@ def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry) -> None:
         super().__init__(hass, _LOGGER, name=DOMAIN, update_interval=UPDATE_INTERVAL)
 
         protocol = "https" if config_entry.data[CONF_SSL] else "http"
+        self.api_url = f"{protocol}://{config_entry.data.get(CONF_HOST)}"
+        self.api_port = config_entry.data.get(CONF_PORT)
```

Comment:
> This integration has quite a good test coverage, can we add test coverage for these lines as well?

The coordinator is completely untested and I'm abit stuck here since I'm not a pytest expert. Maybe someone can point me in a direction?